### ISSUE TITLE
About the result of keyboard key press

### ISSUE THREAD

**Issue by xuchlei** [octalmage/robotjs#578]
## Method : robot.typeString()
* Win10 the content input and outout is same, such as `A` output `A`, `*` output `*`
* Deepin(Debian), CentOS7(RedHat) capital letters is normal, but the characters are not good, such as `A` output `A,` `*` output `8`. the special character `"` cannot output, `<` output `<`

## Method : robot.keyTap(curChar, ["shift"]) used with shift
*  Win10 output is normal
* Deepin(Debian), CentOS7(RedHat) `"` doesn’t work, but you can use `'` plus shift, `<` output `>`

## to sum up
I think there is a problem with characters`"`,`<` in Linux

**Comment by octalmage** [comment]
Closing stale issues as part of a cleanup. If this is still an issue, please reopen and I'll take a look. Thanks!
